Made changes to reflect common Jekyll elements

mcmullengreg opened the pull request:
I implemented an include structure, integrated a more dynamic page title and meta description in the header and cleaned up the SASS/CSS issues that didn't actually change the CSS on `jekyll serve`.

If anything needs updated let me know. I'll be more than happy to.

mcmullengreg committed:
Updated CSS Files to compile properly

Noticed that upon changing files within the sass folder that nothing was actually changing in the stylesheets upon compile
so I modified the files to add the front matter where appropriate, and nearly completely trashed the basic .css files

FontAwesome and Skel are still their basic CSS files, where the others are compiled on jekyll server or on upload to GH Pages.

mcmullengreg committed:
Updated to reflect sass_dir change

mcmullengreg committed:
Added include structure

Added

mcmullengreg committed:
Added include structure and updated various components

head.html: Updated title and description, added base URL
default.html: Added various includes to the page such as head, header, analytics and footer

mcmullengreg committed:
Minor SCSS changes so jekyll will server properly

mcmullengreg committed:
Fixed Build Error

Missed a closing `}}` in one of the prepend statements

mcmullengreg committed:
Strip HTML from site meta description

bcfurtado commented:
I just spend some time working in my PR [https://github.com/CloudCannon/Strata-Jekyll-Theme/pull/7] which does the same thing because I didn't thought already could have some PR to fix this.

:+1: for you PR.

bcfurtado commented:
Actually, I was seeing your changes I realised that you removed **PIE.htc** and **html5shiv.js** files but those files still are been using in the current code. Can you explain me why that?

mcmullengreg committed:
Add IE support files back into the structure

For some reason these files were removed and I have no solid reason why they were.

mcmullengreg commented:
I can't explain why those files were removed. Added back in with 1f6acd1
